proto/controlclient.proto

```diff
@@ -83,3 +85,22 @@ message _FlushCacheRequest {
 message _FlushCacheResponse {
 
 }
+
+message _GenerateTokenRequest {
+  string session_token = 1;
+  // how many seconds do you want the api token to be valid for?
+  uint32 valid_for_seconds = 2;
+  string subject = 3;
```

Comment:
At least for now, I don't think we should have the `subject` be passed as a part of the request - instead the API will read it out of the token used to make the request. That is, you can't create an API token for someone else, only for yourself.

Reply:
>  instead the API will read it out of the token used to make the request

the `token` you are referring to here is the session token?

Reply:
yes, it will be a session token (although for first pass can be from any type of token we already accept, and we'll add the session-only restriction later)